Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessSelectMetadata.java

```diff
@@ -112,11 +112,18 @@ public String getInput() {
     }
 
     @Override
-    public Collection<Metadata> getMetadata() throws InvalidMetadataValueException {
+    public Collection<Metadata> getMetadataWithFilledValues() throws InvalidMetadataValueException {
+        return getMetadata(true);
+    }
+
+    @Override
+    public Collection<Metadata> getMetadata(boolean skipEmpty) throws InvalidMetadataValueException {
         int items = selectedItems.size();
         Collection<Metadata> metadata = new HashSet<>((int) Math.ceil(items / .75));
         String key = settings.getId();
         MdSec domain = DOMAIN_TO_MDSEC.get(settings.getDomain().orElse(Domain.DESCRIPTION));
+        selectedItems.removeAll(Collections.singletonList(""));
+        selectedItems.removeAll(Collections.singletonList(null));
```

Comment:
removeAll should be called only once and with collection containing empty string and null for example or use with filter of stream.